paper/src/main/java/vg/civcraft/mc/civmodcore/world/locations/chunkmeta/WorldChunkMetaManager.java

```diff
@@ -166,7 +164,7 @@ void loadChunk(int x, int z) {
 		}
 		chunkCoord.minecraftChunkLoaded();
 	}
-	
+
 	private void registerRegularSaveRunnable() {
```

Comment:
It may be more efficient here to simply have a task that "boops" the other threadpool to do the work, instead of having a full scheduled executor just for saving.